helpers/integration.go

```diff
@@ -546,3 +548,14 @@ func AddFinalizerToScenario(adapterClient client.Client, logger IntegrationLogge
 
 	return nil
 }
+
+func IsSnapshotfYoungerThanThreshold(snapshot *applicationapiv1alpha1.Snapshot) bool {
+	snapshotCreationTime := snapshot.GetCreationTimestamp().Time
+	durationSinceSnapshotCreation := time.Since(snapshotCreationTime)
+
+	if durationSinceSnapshotCreation < SnapshotRetryTimeout {
+		return true
+	} else {
+		return false
+	}
```

Comment:
```suggestion
	return durationSinceSnapshotCreation < SnapshotRetryTimeout

```

Reply:
Yes agreed, not sure what I was thinking here.